src/components/screens/ScoreSetCreator.vue

```diff
@@ -0,0 +1,2021 @@
+        // _.cloneDeep is needed because the target gene has been frozen.
+        // this.targetGene = _.cloneDeep(this.existingTargetGene)
+        const targetGene = _.cloneDeep(this.existingTargetGene)
+        this.targetGene = _.merge({
+          name: null,
+          category: null,
+          type: null,
+          targetSequence: {
+            sequenceType: null,
+            sequence: null,
+            label: null,
+            taxonomy: null
+          },
+          targetAccession: {
+            assembly: null,
+            accession: null
+          }
+        }, targetGene)
+        this.targetGene.externalIdentifiers = {}
+        for (const dbName of externalGeneDatabases) {
+          this.targetGene.externalIdentifiers[dbName] = (targetGene.externalIdentifiers || [])
+            .find(({ identifier }) => identifier?.dbName == dbName) || {
+            identifier: null,
+            offset: null
+          }
+        }
+      }
+    },
+    item: {
+      handler: function () {
+        this.resetForm()
+      }
+    },
+    itemId: {
+      handler: function () {
+        this.setItemId(this.itemId)
+      },
+      immediate: true
+    },
+    defaultLicenseId: {
+      handler: function () {
+        if (this.licenseId == null) {
+          this.licenseId = this.defaultLicenseId
+        }
+      }
+    },
+    geneName: {
+      handler: async function(newValue, oldValue) {
+        if (newValue == oldValue) {
+          return;
+        }
+        this.geneNameDropdownValue = this.geneName?.name || null
+        if (this.geneNameDropdownValue) {
+          this.geneNameAccessionSuggestions = await this.fetchTargetAccessionsByGene(this.geneNameDropdownValue)
+        }
+      }
+    },
+    assembly: {
+      handler: async function(newValue, oldValue) {
+        if (newValue == oldValue) {
+          return;
+        }
+        this.assemblyDropdownValue = this.assembly?.trim() || null
+        if (this.assemblyDropdownValue) {
+          this.assemblySuggestions = await this.fetchTargetAccessionsByAssembly(this.assemblyDropdownValue)
+        }
+      }
+    },
+  },
+
+  methods: {
+    validateWizardStep: function(step) {
+      switch (step) {
+        case 0: {
+          // Step 0 is valid if
+          // - The score set is a meta-analysis and at least one meta-analyzed score set has been chosen..
+          // - The score set is a superseding score set, and the superseded score set has been chosen.
+          // - Or the score set is neither, and its parent experiment has been chosen.
+          return !!(this.isMetaAnalysis && this.metaAnalyzesScoreSets.length > 0)
+              || !!(this.isSupersedingScoreSet && this.supersededScoreSet)
+              || !!(!this.isMetaAnalysis && !this.isSupersedingScoreSet && this.experiment)
+        }
+        case 1: {
+          return this.title && this.shortDescription && this.abstractText && this.methodText
+        }
+
```

Comment:
Per the comment above, we could minimally add a check on the length of the user added target genes list to ensure they've added at least one before advancing:
```suggestion
case 2: {
      return this.targetGenes.length > 0
}
```